src/Network/Network/AzureFirewall/NewAzureFirewallCommand.cs

```diff
@@ -99,10 +99,10 @@ public class NewAzureFirewallCommand : AzureFirewallBaseCmdlet
         public PSVirtualNetwork VirtualNetwork { get; set; }
 
         [Parameter(
-            Mandatory = true,
+            Mandatory = false,
             ValueFromPipelineByPropertyName = true,
             ParameterSetName = "IpConfigurationParameterValues",
-            HelpMessage = "One or more Public IP Addresses. The Public IP addresses must use Standard SKU and must belong to the same resource group as the Firewall.")]
+            HelpMessage = "Allowing zero data Public IP for FT. The Public IP addresses must use Standard SKU and must belong to the same resource group as the Firewall.")]
```

Comment:
The original help message is still accurate - the value of this parameter should be One or more PIPs